src/runtime-rs/crates/hypervisor/src/lib.rs

```diff
@@ -28,6 +29,7 @@ const VM_ROOTFS_DRIVER_BLK: &str = "virtio-blk";
 const VM_ROOTFS_DRIVER_PMEM: &str = "virtio-pmem";
 
 pub const HYPERVISOR_DRAGONBALL: &str = "dragonball";
+pub const HYPERVISOR_QEMU: &str = "qemu";
```

Comment:
It's better to add a blank line after line 32.

Reply:
True, fixed.